hms-lambda-handler/pom.xml

```diff
@@ -128,21 +128,6 @@
             <artifactId>commons-configuration2</artifactId>
             <version>2.8.0</version>
         </dependency>
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>31.0.1-jre</version>
-        </dependency>
-        <dependency>
-            <groupId>com.fasterxml.woodstox</groupId>
-            <artifactId>woodstox-core</artifactId>
-            <version>6.2.8</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-configuration2</artifactId>
-            <version>2.7</version>
-        </dependency>
```

Comment:
This part is duplicated, so moving old versions. newer versions are from L116 to L130.